addons/account/models/account_move.py

```diff
@@ -1496,6 +1507,13 @@ def _compute_amount(self):
                 if reverse_moves_full_recs.mapped('reconciled_line_ids.move_id').filtered(lambda x: x not in (reverse_moves + reverse_moves_full_recs.mapped('exchange_move_id'))) == move:
                     new_pmt_state = 'reversed'
 
+            # Dynamically check if the partner's credit limit is exceeded, and display a warning accordingly
+            if move.partner_credit <= move.partner_credit_limit:
+                partner_updated_credit = move.partner_credit + move.amount_total
```

Comment:
be careful amount_total is not express in the company currency. You should convert amount_total in the company currency.

Reply:
Indeed, this is why (among other things) it was decided to leave that part aside for the moment, and only focus on the Total Receivable amount.